src/mscorlib/src/System/Reflection/RuntimeModule.cs

```diff
@@ -464,7 +464,7 @@ public override Type GetType(String className, bool throwOnError, bool ignoreCas
 
             RuntimeType retType = null;
             Object keepAlive = null;
-            GetType(GetNativeHandle(), className, throwOnError, ignoreCase, JitHelpers.GetObjectHandleOnStack(ref retType), JitHelpers.GetObjectHandleOnStack(ref keepAlive));
+            GetType(GetNativeHandle(), className, ignoreCase, throwOnError, JitHelpers.GetObjectHandleOnStack(ref retType), JitHelpers.GetObjectHandleOnStack(ref keepAlive));
```

Comment:
This is not the right fix. The right fix is to swap the argument names in the GetType declaration at the top of the file. 

There is no actual functionality bug here. It is just a cosmetic issue that the declaration has the argument names swapped.